### PR TITLE
PHP CheatSheet: Update the cheat sheet to make it more relevant

### LINKED ISSUE #3515: PHP Cheat Sheet: missing functions
Some feedback from a DDG user:

> no often used builtin php functions mentioned here

What else can be added to make this more useful?

------
IA Page:  http://duck.co/ia/view/php_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @sahildua2305

### PR DESCRIPTION
**Pull request by sahildua2305**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [x] Bug fix: **`{IA Name}: Fix #3515`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Provide an overview of the changes this pull request introduces.

###### Which issues (if any) does this fix?

Fixes #3515 

###### People to notify (@mention interested parties)
@edgesince84 @moollaza 

---

Instant Answer Page: https://duck.co/ia/view/php_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by davidofferman** [comment]
I have a few suggestions. I've never made a cheatsheet yet, so I'm just posting the ideas here. If you are busy, I can try doing this and send a PR to you.

- [ ] Add sort functions to their own group, outside of the "PHP Array Functions" group.
- [ ] Hashing Functions: `md5`, `sha1`, `hash`, `hash_file`, `hash_equal`.
- [ ] Date/Time Functions: `time`, `date`, `mktime`, `microtime`

Determining what's important or most often used is difficult, but after the changes you made and the above few suggested functions, it's a good starting point.

File: share/goodie/cheat_sheets/json/php.json [hunk 1]
```diff
@@ -7,6 +7,8 @@
         "sourceName": "PHP Manual",
         "sourceUrl" : "http://php.net/manual/en/index.php"
     },
+
+    "columns": 2,
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/php.json]: this won't be necessary because it's using the code template 👍

Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/php.json]: Ah! I forgot. Thanks.

**Comment by sahildua2305** [comment]
Thanks for your inputs @davidofferman. These are really valuable changes. :heart_eyes_cat:

File: share/goodie/cheat_sheets/json/php.json [hunk 2]
```diff
@@ -42,57 +47,54 @@
         }],
         "PHP Array Functions": [{
             "val": "count all elements in array",
-            "key": "count()"
+            "key": "count($array)"
+        }, {
+            "val": "retrieve all the values from an associative array",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/php.json]: Lets make the capitalization of first letter in the val field consistent

Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/php.json]: Thanks @jophab! Done :)

**Comment by sahildua2305** [comment]
@moollaza I think we can go ahead with this. :+1:

**Comment by moollaza** [comment]
@sahildua2305 Thanks! This LGTM :+1:

**Pull request merged**